spec/lita/handlers/totems_spec.rb

```diff
@@ -123,6 +128,40 @@ def send_destroy_message
         end
       end
 
+      context "with a timeout" do
+        it "includes the timeout in the totems' info" do
+          Timecop.freeze("2014-03-01 12:00:00") do
+            send_message("totems add noir timeout: 10", as: carl)
+            send_message("totems add noir", as: another_user)
+            send_message("totems add noir timeout: 20", as: yet_another_user)
+          end
+          Timecop.freeze("2014-03-01 13:00:00") do
+            send_message("totems info noir")
+            expect(replies.last).to eq <<-END
+1. Carl (held for 1h) - timeout: 10
+2. person_1 (waiting for 1h) - timeout: 24
+3. person_2 (waiting for 1h) - timeout: 20
+            END
+          end
+        end
+
+        it "triggers the timeout when the totem is in the list" do
+          send_message("totems add noir timeout: 3", as: carl)
+          wait(10).for do
+              send_message("totems info noir")
+              replies.last.empty?
+          end.to be(true)
+        end
+
+        it "doesn't trigger the timeout on any totem" do
```

Comment:
nit: Maybe rename to doesn't trigger timeout when the totem is not a demo environment. Unless you think that list will change to have a meaning outside of "demo"